daemon/src/setup_contract.rs

```diff
@@ -459,7 +459,7 @@ pub async fn roll_over(
         .context("Empty stream instead of Msg2")?
         .try_into_msg2()?;
 
-    let revoked_commit = finalize_revoked_commits(&dlc, own_cfd_txs.commit.1, msg2.into())?;
+    let revoked_commit = finalize_revoked_commits(&dlc, dlc.commit.1, msg2.into())?;
```

Comment:
:wrench: This `roll_over` function is not used anymore. Just remove it? 

Maybe you meant to modify the deprecated version? I wouldn't bother because it would be a breaking change for a feature that is not implemented fully. And the code should disappear soon.

Reply:
Oh my, my effort to re-use code in that function was totally useless because it is unused 😅

I opened a separate PR #2297 to remove the function and based this PR on it.